chainguard-dev/edu
# Updating image reference docs with new menu organization

### What should this PR do?
This manual PR uses content generated from the new autodocs, and updates the image reference docs to the new menu structure. The change has a dramatic impact on build time and memory consumption when building the website. Images now use tabs to organize the contents.

Includes an Nginx redirect (🤞🏼):

```
"~^/chainguard/chainguard-images/reference/(.+)?/overview/(.+)?$" /chainguard/chainguard-images/reference/$1/;
```

Following, I will disable the current workflow running in this repository and enable the new autodocs workflow in the [images-autodocs](https://github.com/chainguard-dev/images-autodocs) repository, which will continue creating pull requests here on a daily basis.

### How should this PR be tested?
Use the [preview link](https://deploy-preview-995--ornate-narwhal-088216.netlify.app/chainguard/chainguard-images/reference/apko/) to check the left menu and image reference pages.
